go/enclave/rollupchain/rollup_chain.go

```diff
@@ -342,7 +309,7 @@ func (rc *RollupChain) updateL1State(block types.Block, receipts types.Receipts,
 	// We check whether we've already processed the block.
 	_, err := rc.storage.FetchBlock(block.Hash())
 	if err == nil {
-		return rc.rejectBlockErr(common.ErrBlockAlreadyProcessed)
```

Comment:
Here and below, I avoided using `rc.rejectBlockErr`. I just wrap the error once over in `enclaveImpl.SubmitL1Block` now, before it gets returned.